packages/node_modules/nav-frontend-etiketter/src/index.tsx

```diff
@@ -12,7 +12,7 @@ const cls = (type, className) => classNames('etikett', className, {
 });
 
 export interface EtikettProps {
-    typo: string;
+    typo?: string;
```

Comment:
`EtikettProps` bør egentlig ha typo som required, men sette defaultProp til `normaltekst` statisk på klassen (nå blir den satt i uthentinga av verdien fra props).

Altså:
`static defaultProps = { typo: 'normaltekst' }`

Reply:
Samme som den andre `typo` er ikke isRequired på prop-types fra før. Altså er det mismatch mellom TS og JS-varianten. Hvor TS i dag har den som required, men JS-verianten har den som optional i prop-types. Å endre den til å være required er en breaking-change, så tenker vi burde beholde den som optional.

Reply:
Gir det noe mening å ha en default-verdi hvis prop er required?

Reply:
Kan ikke fjerne assignment til typo på linje 33, siden TypeScript ikke helt forstår seg på propTypes og defaultProps. Så for at typescript skal forstå at den har en defaultverdi må den settes. (for hvis ikke den blir assignet får man klage på at `TypografiBase` skal ha inn `type` og den kan ikke være `undefined`. En mulighet er å endre til å ha følgende:
`type={typo!}` på `typegrafibase`. Altså si ifra til TS om at typo alltid har en verdi. Er dette mer ønskelig?